app/views/deploys/new.html.erb

```diff
@@ -18,6 +18,8 @@
         </div>
       <% end %>
 
+      <%= Samson::Hooks.render_views(:before_deploy_form, self, project: @project, form: form) %>
```

Comment:
Some comment in hook.rb like `# for external plugins` so nobody removes it in an effor to clean up unused hooks could be good too :)